docs/UserGuide.md

```diff
@@ -129,6 +129,20 @@ java -version
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+
+### Task parameters
```

Comment:
Not sure how Jekyll is going to display a header within the blue box. Perhaps it may be better if you put task parameters in a separate blue box of its own. So there's one box for general command format and another specifically for task parameters.

Reply:
Or don't put task params in a blue box, since the header might look out of place.